### ISSUE TITLE
[rate-limit] deny status set to 0

### ISSUE THREAD

**Issue by bedis** [haproxytech/kubernetes-ingress#25]
in the configuration generated by rate-limit feature, the deny status code is set to 0 in the generated configuration:

```
  acl ratelimit_cnt_abuse src_get_gpc0(RateLimit) gt 0
  acl ratelimit_inc_cnt_abuse src_inc_gpc0(RateLimit) gt 0
  acl ratelimit_is_abuse src_http_req_rate(RateLimit) ge 10
  http-request deny deny_status 0 if ratelimit_is_abuse ratelimit_inc_cnt_abuse
  http-request deny deny_status 0 if ratelimit_cnt_abuse
```

**Comment by bedis** [comment]
here is the relevant PR: https://github.com/haproxytech/kubernetes-ingress/pull/26

**Comment by bedis** [comment]
Fixed with commit https://github.com/haproxytech/kubernetes-ingress/commit/880b515922a8a768332feb07dcb313893c407054
Available since version 1.1.2
